src/app/effects/work-item.effects.ts

```diff
@@ -80,13 +83,14 @@ export class WorkItemEffects {
           wItem.createId = createID;
           return wItem;
         })
+        .switchMap(w => util.workitemMatchesFilter(this.route.snapshot, this.filterService, this.workItemService, w))
```

Comment:
Why are you passing the filterService and workItemService here? You can directly import it into util and use it.

Reply:
@vikram-raj To inject the services in utill i would need to create a class for this and I only needed services for this method only. So, passed the instances of these services through method call.

Reply:
@sahil143 Ok got it.